paimon-core/src/main/java/org/apache/paimon/catalog/FileSystemCatalog.java

```diff
@@ -122,6 +123,9 @@ protected void dropTableImpl(Identifier identifier) {
     @Override
     public void createTableImpl(Identifier identifier, Schema schema) {
         Path path = getDataTableLocation(identifier);
+        if (schema.options().containsKey(AUTO_CREATE.key())) {
+            schema.options().put(AUTO_CREATE.key(), String.valueOf(Boolean.FALSE));
```

Comment:
Can we just throw exception here?

Reply:
https://github.com/apache/incubator-paimon/blob/fdcc24f90f2762b294ac9ac2270527d6b3de5c95/paimon-core/src/main/java/org/apache/paimon/catalog/AbstractCatalog.java#L176


Should we handle it here，  Setting `auto-create = true` is useless